apollo-core/src/main/java/com/ctrip/framework/apollo/core/ApolloClientSystemConsts.java

```diff
@@ -146,4 +146,14 @@
    * enable property names cache environment variables
    */
   public static final String APOLLO_PROPERTY_NAMES_CACHE_ENABLE_ENVIRONMENT_VARIABLES = "APOLLO_PROPERTY_NAMES_CACHE_ENABLE";
+
+  /**
+   * enable property names cache
+   */
+  public static final String APOLLO_FILE_CACHE_ENABLE = "apollo.file.cache.enable";
```

Comment:
Is it possiable exists more cache in client in the future? 

Maybe we can plan the orgnaization of key, let it more extensible.

* apollo.cache.file.enable
* apollo.cache.config-map.enable
* apollo.cache.redis.enable

Use `apollo.cache.file.enable` or other key to instead of `apollo.file.cache.enable`?

Reply:
I agree `apollo.cache.file.enable` is better considering future extensibility.

Reply:
fixed